### PR TITLE
coda.io instead of coda.com

### PR DESCRIPTION
**Pull request by mariaremote**
found a broken link, and fixed it! (they have changed domain)
Thanks :)

### CODE REVIEW

File: docs/integrations/builtin/credentials/coda.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You can use these credentials to authenticate the following nodes with Coda.
 
 ## Prerequisites
 
-Create a [Coda](https://www.coda.com/) account.
+Create a [Coda](https://www.coda.io/) account.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/coda.md:15]:
```suggestion
Create a [Coda](https://www.coda.io/){:target=_blank .external-link} account.
```

Comment (StarfallProjects) [reply docs/integrations/builtin/credentials/coda.md:15]: Open external links in new tab. I realise it didn't have this before - this is something we're fixing as we see instances of it.

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Link in new tab (see suggestion)

File: docs/integrations/builtin/credentials/coda.md (hunk 1 above)
Comment (mariaremote) [reply docs/integrations/builtin/credentials/coda.md:15]: Oh yeah you are totally right! I didn't think of adding it, but will now remember for sure 😉

**Review by StarfallProjects — APPROVED** [review]

**Pull request merged**